packages/cli/src/util/__tests__/getHttpOperations.spec.ts

```diff
@@ -4,54 +4,44 @@ describe('getHttpOperationsFromSpec()', () => {
   describe('ref resolving fails', () => {
     it('fails with exception', () => {
       return expect(
-        getHttpOperationsFromSpec(
-          JSON.stringify({
-            openapi: '3.0.0',
-            paths: { $ref: 'abc://' },
-          })
-        )
-      ).rejects.toThrow(
-        /^There's been an error while trying to resolve external references in your document: Error: EISDIR: illegal operation on a directory, read$/
-      );
+        getHttpOperationsFromSpec({
+          openapi: '3.0.0',
+          paths: { $ref: 'abc://' },
+        })
+      ).rejects.toThrow('Unable to resolve $ref pointer "abc://"');
```

Comment:
updated these tests to not stringify the schema object and just pass in the object as the dereference method expects. See comment https://github.com/stoplightio/prism/pull/1189/files#r428930281.